src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, name_prefix='clitest.rg',
                  parameter_name_for_location='resource_group_location', location='westus',
                  dev_setting_name='AZURE_CLI_TEST_DEV_RESOURCE_GROUP_NAME',
                  dev_setting_location='AZURE_CLI_TEST_DEV_RESOURCE_GROUP_LOCATION',
-                 random_name_length=75, key='rg'):
```

Comment:
Agree with @qwordy , please do not modify the framework as it may break other command modules.

Reply:
Made changes without touching framework.